shared/src/jvmMain/kotlin/com/mbta/tid/mbta_app/model/Dependency.jvm.kt

```diff
@@ -0,0 +1,3 @@
+package com.mbta.tid.mbta_app.model
```

Comment:
question(non-blocking): Why is this & FeaturePromo.jvm.kt now needed?

Reply:
An `expect` with no matching `actual` is an error at compile time.